Makefile

```
@@ -179,6 +179,10 @@ test-e2e:
 test-e2e-release:
 	PASSES="build release e2e" ./test.sh $(GO_TEST_FLAGS)
 
+.PHONY: test-release
+test-release:
+	PASSES="release_tests" VERSION="3.7.99" ./test.sh $(GO_TEST_FLAGS)
```

This doesn't work locally because of this environment variable. We need to drop it.

```suggestion
	PASSES="release_tests" ./test.sh $(GO_TEST_FLAGS)
```